src/webln/NostrWeblnProvider.ts

```diff
@@ -115,11 +120,16 @@ export class NostrWebLNProvider implements WebLNProvider {
     return getPublicKey(this.secret);
   }
 
-  signEvent(event: UnsignedEvent) {
+  getPublicKey(): Promise<string> {
+    return Promise.resolve(this.publicKey);
+  }
+
+  signEvent(event: UnsignedEvent): Promise<Event> {
```

Comment:
as you change the signature of this method you also need to adjust the methods where this is used.

Reply:
There are no usages outside of this class (apart from being a breaking change for dependencies)

Reply:
https://github.com/getAlby/alby-js-sdk/blob/master/src/webln/NostrWeblnProvider.ts#L187

Reply:
@bumi sorry about that 😱